lib/java/com/google/android/material/floatingactionbutton/ExtendedFloatingActionButton.java

```diff
@@ -217,6 +234,21 @@ protected void onAttachedToWindow() {
     return behavior;
   }
 
+  /**
+   * Determines how the extended button is hidden when it needs to be collapsed
+   * @param isHide If true, none of the button will be visible when collapsed. If false, the button
+   * will "shrink" and hide button text but keep the icon.
+   *
+   * This option is "false" and perform a shrink by default.
+   */
+  public void setHideBehavior(boolean isHide) {
```

Comment:
A client could call `getBehavior()` and then call these methods directly. Do you think there's value in adding this wrapper method? Or could we just add documentation?

Reply:
Actually... they can't. ExtendedFabBehavior is marked protected so the ExtendedFabBehavior methods cannot be called. This created a need for a client method to be exposed.
 I would not recommend changing the class to public since it is just this one case that needs to be addressed. It would be unnecessary to expose an entire separate class to call... well, one method

Reply:
However, if you want that to happen I would be glad implementing that.

I am just a contributor and I want to make API design decisions that match your needs.

Reply:
'getBehavior' returns 'Behavior<ExtendedFloatingActionButton>', doesn't it? A client can't cast that to protected 'ExtendedFloatingActionButton.ExtendedFloatingActionButtonBehavior' in order to get to the properties without extending from 'ExtendedFloatingActionButton' or somthing. Or am I missing something?

Reply:
You are missing nothing @falsevapor. That is the exact conversation we are having!

Reply:
@ethanhsuhsu, sorry, I missed your relies here. Somehow github just showed the message from @cketcham only... nevermind

Reply:
Yeah.. the fact that that class is protected is what I was missing. I guess it makes sense for it to be protected since it's very specific to Extended Fab..